docs/reference/rollup/apis/rollup-api.asciidoc

```diff
@@ -53,8 +53,9 @@ POST /my-index-000001/_rollup/rollup-my-index-000001
 
 * You can only roll up an index that contains:
 
-** A <<date,`date`>> or <<date_nanos,`date_nanos`>> timestamp field
-** One or more <<number,numeric>> fields
+** A <<date,`date`>> or <<date_nanos,`date_nanos`>> timestamp field.
+** At least one <<number,numeric>>, `date`, or `date_nanos` field other than the
```

Comment:
should we highlight what this additional field would be used for? as in, why it is required: metrics.